lib/argument.js

```diff
@@ -33,7 +33,7 @@ class Argument {
         break;
     }
 
-    if (this._name.length > 3 && this._name.slice(-3) === '...') {
+    if (this._name.length > 3 && this._name.endsWith('...')) {
```

Comment:
The slice was a bit unusual, and the length test is redundant too.

```suggestion
    if (this._name.endsWith('...')) {
```